JavaAssignments/src/saket/Bank.java

```diff
@@ -0,0 +1,63 @@
+package saket;
+
+import java.util.Scanner;
+
+public class Bank {
+
+	static int takeuserinput() {
+
+		Scanner sc = new Scanner(System.in);
+		Scanner sc1 = new Scanner(System.in);
+		Scanner sc2 = new Scanner(System.in);
```

Comment:
Here what is need of three scanner class only one is sufficient to get user input